packages/11ty/_includes/components/figure/soundcloud/element.js

```diff
@@ -0,0 +1,26 @@
+const { html } = require('~lib/common-tags')
+const chalkFactory = require('~lib/chalk')
+
+const { error } = chalkFactory('Figure Video')
+
+module.exports = function (eleventyConfig) {
+  return function ({ id, media_id: mediaId }) {
+    if (!mediaId) {
+      error(`Cannot render SoundCloud component without 'media_id'. Check that figures data for id: ${id} has a valid 'media_id'`)
+      return ''
+    }
+
+    const src = `https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/${mediaId}`
+
+    return html`
+      <iframe
+        allow="autoplay"
+        frameborder="no"
+        height="166"
+        scrolling="no"
+        src="${src}&auto_play=false&color=%23ff5500&hide_related=true&show_comments=false&show_reposts=false&show_teaser=false&show_user=false"
```

Comment:
The long string of parameters can be difficult to maintain should any of the values change, creating a `params` object with the key:value for each parameter and using the `URLSearchParams` object to create the string may be more explicit and easier to read.

Reply:
I refactored this in `figure/audio/element.js` to use `URL`/`URLSearchParams` to build the iframe `src`